src/controllers/crate_owner_invitation.rs

```diff
@@ -139,7 +139,16 @@ fn prepare_list(
                 )
                 .load(&*conn)?
         }
-        Page::Numeric(_) => unreachable!("page-based pagination is disabled"),
+        Page::Numeric(_) => query
+            .pages_pagination(pagination)
+            .filter(
+                crate_owner_invitations::crate_id.gt(seek_key.0).or(
+                    crate_owner_invitations::crate_id
+                        .eq(seek_key.0)
+                        .and(crate_owner_invitations::invited_user_id.gt(seek_key.1)),
+                ),
+            )
+            .load(&*conn)?,
```

Comment:
I think page-based pagination has been explicitly not implemented here for performance reasons. We would like to move all of our APIs to seek-based pagination in the future for these reasons.

Reply:
Got it! Do we have support for seek-based pagination already in some place?

Reply:
@Turbo87 I can see Seek pagination is already supported for owner invites for the backend here: https://github.com/rust-lang/crates.io/blob/04d4350b3c2b561a39e14abdec7c2d1e727eaf2c/src/controllers/crate_owner_invitation.rs#L130

But I still have to check if it works for the pagination implementation